Sources/JavaKitMacros/JavaFieldMacro.swift

```diff
@@ -79,4 +84,18 @@ extension JavaFieldMacro: AccessorMacro {
 
     return accessors
   }
+
+  private static func isInStaticContext(context: some MacroExpansionContext) -> Bool {
```

Comment:
```suggestion
  private static func isInJavaClassContext(context: some MacroExpansionContext) -> Bool {
```

maybe? Not to be confused in being in some actual `static` etc.

Reply:
updated to use java class in this. I can see how static could be confusing since it meant static in the `java` form not the `swift`